src/link/output.c

```diff
@@ -273,6 +275,47 @@ static struct SortedSection const **nextSection(struct SortedSection const **s1,
 	return (*s1)->section->org < (*s2)->section->org ? s1 : s2;
 }
 
+// Checks whether a symbol is
+static bool canStartSymName(char c)
+{
+	return (c >= 'A' && c <= 'Z') || (c >= 'a' && c <= 'z') || c == '_';
+}
+
+static void printSymName(char const *name)
+{
+	for (char const *ptr = name; *ptr != '\0'; ) {
+		char c = *ptr;
+
+		if ((c >= 'A' && c <= 'Z') || (c >= 'a' && c <= 'z') || (c >= '0' && c <= '9')
+		 || c == '_' || c == '@' || c == '#' || c == '$' || c == '.') {
+			fputc(c, symFile);
+		++ptr;
```

Comment:
```suggestion
			++ptr;
```

Reply:
And they tell me `clang-format` is not useful :smirk:

Reply:
What's most useful to me about Prettier with Typescript is that I *don't* have to type `prettier the/file.ts<Enter>` every time I save. :P (Okay, that's VS Code + the plugin.)

Reply:
Nor do I have to with Sublime's `clang-format` or `rustfmt` plugins!